### ISSUE TITLE
Not communicating with R

### ISSUE THREAD

**Issue by juanmahvelez** [randy3k/R-Box#29]
Hi,

I am testing lines of code from R-Box. I am on Mac 10.8, R version 3.1.2. The syntax highlighting works fine, but when I try to run code using cmd+return, nothing happens. I've made no changes to the default preference settings. I tried to remove and uninstall the R-Box package and reinstall but the issue persists.

**Comment by randy3k** [comment]
Are you running R on the terminal or using the R GUI? The default target is the R GUI. If you want to switch between different applications. You should run the `R-Box Application` command in the command palette.

**Comment by juanmahvelez** [comment]
I am trying to run from the R GUI. It is not opening. When I run the `R-Box Application` command, nothing changes. Even if I select terminal, cmd+return still will not launch R.

**Comment by randy3k** [comment]
That's strange. Did you bind `cmd+enter` to other commands?

**Comment by juanmahvelez** [comment]
No. In fact, just to be sure, I just uninstalled Sublime Text 2, deleted all of my Sublime Folders from ~/Library/Application Support/, re-downloaded and reinstalled Sublime, reinstalled R-Box, and tried again. Still does not work. Have you tested with R v3.1.2?

**Comment by juanmahvelez** [comment]
Update: I updated R to 3.1.2 on my windows machine and R-Box works fine there. It just doesn't work on my Mac.

**Comment by randy3k** [comment]
Yes. I am using Mac and I confirm that it is still working...
Did your console report anything? Using ctrl + ` to open it

**Comment by juanmahvelez** [comment]
My console reports nothing when I use cmd + return to run R code using R-Box.

**Comment by juanmahvelez** [comment]
I again tried installing R-Box, this time while viewing my console for errors. For all scripts that contain `from .misc import RBoxSettings`, such as completions.py, misc.py, etc., there is this error:

> Traceback (most recent call last):
>   File "./sublime_plugin.py", line 62, in reload_plugin
>   File "./completions.py", line 6, in <module>
>     from .misc import RBoxSettings
> ValueError: Attempted relative import in non-package

**Comment by randy3k** [comment]
Oh, are you using subline text 2?

**Comment by juanmahvelez** [comment]
Yes

**Comment by juanmahvelez** [comment]
I'm assuming that by your last question you were suggesting that I should be using Sublime Text 3. I installed Sublime Text 3 and it now works. I still don't understand why it's not working through Sublime 2.

**Comment by randy3k** [comment]
Did you install R-Box to Sublime Text 2 via Package control? You should be able to get a version work with Sublime Text 2 via Package control. If you clone this repo directly to your Sublime Text 2 package directory, it will throw an error similar to what you got. It is because there are some compatibility issues between python 2 and python 3 which drive Sublime Text 2 and 3 respectively.

**Comment by juanmahvelez** [comment]
I used Package Control, but I had to add the github repo url to the package directory because it did initially not appear in the list of available packages to install through Package Control. I'm going to close this issue because the title is now misleading, since it turns out my installation of R-Box in Sublime Text 2 failed. You've been very helpful. I appreciate your taking the time to answer these questions as well as your awesome contributions, Enhanced-R and R-Box.
